### DISCUSSION TITLE
ArgoCD Backend

### DISCUSSION THREAD

**Discussion by jkroepke** [glasskube/glasskube#243] [Ideas]
**Is your feature request related to a problem? Please describe.**
I would like to use glasskube together with ArgoCD.

**Describe the solution you'd like**
Instead glasskube apply manifest, glasskube could apply a single ArgoCD Application and ArgoCD may take care of apply the manifests. ArgoCD als natively supports helm and compared to helm, it can handle CRD upgrades.

This is useful to observe and ensure the desired state.

**Describe alternatives you've considered**
N/A

**Additional context**

**Comment by pmig** [comment]
Thanks @jkroepke for raising the feature request. As this is something we have no immediate plans on getting started it is also not something we want to rule out. In order to get more feedback from the community on this topic I am converting this issue into a discussion, so everyone can state their opinion.
